app/assets/javascripts/shared/dradis_datatable.js

```diff
@@ -63,9 +95,84 @@ class DradisDatatable {
     this.unbindDataTable();
   }
 
+  handleBulkDeleteSuccess(rows, data) {
+    // remove() will remove the row internally and draw() will
+    // update the table visually.
+    rows.remove().draw();
+    this.showBulkDeleteBtn(false);
+
+    if (data.success) {
+      if (data.jobId) {
+        // background deletion
+        this.showConsole(data.jobId);
+      } else {
+        // inline deletion
+        this.showAlert(data.msg, 'success');
+      }
+    } else {
+      this.showAlert(data.msg, 'error');
+    }
+  }
+
+  handleBulkDeleteError(rows) {
+    rows.nodes().toArray().forEach(function(tr) {
+      tr.querySelectorAll('td')[2].innerHTML = '<span class="text-error">Please try again</span>';
+    })
+  }
+
+  showAlert(msg, klass) {
+    this.$table.parent().find('.alert').remove();
+
+    this.$table.parent().prepend(`
+      <div class="alert alert-${klass}">
+        <a class="close" data-dismiss="alert" href="javascript:void(0)">x</a>
+        ${msg}
+      </div>
+    `);
+  }
+
+  showBulkDeleteBtn(boolean) {
```

Comment:
```suggestion
  toggleBulkDeleteBtn(isShown) {
```

We're not only showing it here but hiding it too.